From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Jung-Kwang Lee <jungkwang.lee@softcamp.co.kr>
Date: Mon, 9 Feb 2026 17:45:53 +0900
Subject: Allow Electron visibility for CDM renderer

Electron's renderer clients use components/cdm/renderer to register Widevine
key-system support. Allow //electron visibility for this embedder-specific
integration.

diff --git a/components/cdm/renderer/BUILD.gn b/components/cdm/renderer/BUILD.gn
index fbe2ec2264cd170fe9c5ec9bcedf9fdd4732a2a0..614bb127acc01f119077f98858bd74e4d8489da8 100644
--- a/components/cdm/renderer/BUILD.gn
+++ b/components/cdm/renderer/BUILD.gn
@@ -14,6 +14,7 @@ static_library("renderer") {
     "//components/cdm/*",
     "//content/shell/*",
     "//fuchsia_web/*",
+    "//electron/*",
   ]
 
   sources = [
